package.json

```diff
@@ -6,10 +6,11 @@
   },
   "version": "1.0.0",
   "description": "Save Microsoft Stream videos for offline enjoyment.",
-  "main": "destreamer.js",
+  "main": "build/src/destreamer.js",
+  "bin": "build/src/destreamer.js",
   "scripts": {
     "build": "echo Transpiling TypeScript to JavaScript... & node node_modules/typescript/bin/tsc --listEmittedFiles",
-    "run": "node ./destreamer.js",
+    "run": "node ./build/src/destreamer.js",
```

Comment:
Can we get rid of that `src/` in the middle?
`./build/*.js`, we probably need to be more specific in how we call `tsc`.

We could have
```
`./build/*.js` or `./build/app/*.js` for the app itself
`/build/test/*.js` for tests

Reply:
can you explain?
isn't run supposed to run destreamer.js?
other files are not "executables"

mocha will call test automatically, we only need to set the path to a folder that contains the test folder

Reply:
No, my problem is just with the naming, the `src`. Those files aren't the source, they are the result of the build.. the "binaries".

Reply:
ok, i don't how to tell ts to put files in build folder

Reply:
I'll merge as is, i'll look that up in TypeScript's doc (said no engineer ever :))